### PR TITLE
Only remove FEC if videofec=false.

### PR DESCRIPTION
**Pull request by stefanholmer**
**Description**

**Purpose**

### CODE REVIEW

File: src/web_app/js/sdputils.js [hunk 1]
```diff
@@ -247,7 +247,10 @@ function removeCodecByPayloadType(sdpLines, payloadType) {
   return sdpLines;
 }
 
-function maybeRemoveVideoFec(sdp) {
+function maybeRemoveVideoFec(sdp, params) {
+  if (params.videoFec === 'true')
```
Comment (KaptenJansson) [inline src/web_app/js/sdputils.js]: please use curly braces (linter will complain ;))

**Comment by stefanholmer** [comment]
Not sure why travis complains, it doesn't complain locally?

**Comment by KaptenJansson** [comment]
Seems like we have managed to check in a lint error, https://github.com/stefanholmer/apprtc/blob/cbd5fe6c75bea91783c88f4406f059c6f5b500cc/src/web_app/js/sdputils.js#L51 These should be indented like the switch case above.

**Comment by KaptenJansson** [comment]
I will fix the lint errors separately.

**Pull request merged**